examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/VectorDataFrameUdfs.cs

```diff
@@ -62,9 +62,9 @@ private static FxDataFrame CountCharacters(FxDataFrame dataFrame)
         {
             int characterCount = 0;
 
-            var characterCountColumn = new PrimitiveDataFrameColumn<int>("nameCharCount");
-            var ageColumn = new PrimitiveDataFrameColumn<int>("age");
-            ArrowStringDataFrameColumn nameColumn = dataFrame["name"] as ArrowStringDataFrameColumn;
+            var characterCountColumn = new Int32DataFrameColumn("nameCharCount");
+            var ageColumn = new Int32DataFrameColumn("age");
+            ArrowStringDataFrameColumn nameColumn = dataFrame.Columns.GetArrowStringColumn("name");
```

Comment:
Question on the API. Does the user need to know about `Arrow` here? Can I just get a `String` column?

Reply:
It's because we have 2 string columns: a `StringDataFrameColumn` and an `ArrowStringDataFrameColumn`. Maybe we should've named them `UTF16DataFrameColumn` and `UTF8DataFrameColumn`! `GetArrowStringColumn` returns the `ArrowStringDataFrameColumn` while `GetStringColumn` would return `StringDataFrameColumn`.

Reply:
Got it, thanks. +1 for `UTF16DataFrameColumn` and `UTF8DataFrameColumn`

Reply:
cc @eerhardt on the column names. Maybe something to consider for the future?

Reply:
I think just `StringDataFrameColumn` should stay `StringDataFrameColumn`, since `string` is the common .NET type.

But I could imagine renaming `ArrowStringDataFrameColumn` to `Utf8StringDataFrameColumn`.